src/uwtools/tests/config/formats/test_fieldtable.py

```diff
@@ -1,39 +1,48 @@
-# pylint: disable=missing-function-docstring
+# pylint: disable=missing-function-docstring,redefined-outer-name
 """
 Tests for uwtools.config.formats.fieldtable module.
 """
 
+from pytest import fixture, mark
+
 from uwtools.config.formats.fieldtable import FieldTableConfig
 from uwtools.tests.support import fixture_path
 from uwtools.utils.file import FORMAT
 
 # Tests
 
 
-def test_get_format():
+@fixture(scope="module")
+def config():
+    return fixture_path("FV3_GFS_v16.yaml")
+
+
+@fixture(scope="module")
+def ref():
+    with open(fixture_path("field_table.FV3_GFS_v16"), "r", encoding="utf-8") as f:
+        return f.read().strip()
+
+
+def test_fieldtable_get_format():
     assert FieldTableConfig.get_format() == FORMAT.fieldtable
 
 
-def test_get_depth_threshold():
+def test_fieldtable_get_depth_threshold():
```

Comment:
Here and below, tests for `Config` subclasses now include the format name in their function name, for easier selection with `pytest -k`. For example `pytest -k test_fieldtable`, run from the `src/` directory, will run all the tests in this module.